### ISSUE TITLE
can't create supplier without existing supplier_category

### ISSUE THREAD

**Issue by vali-um** [foodcoops/foodsoft#790]
Hi,

I've set up a new development instance for foodsoft and encountered the issue that i couldn't create a new supplier.
i tracked down the issue to https://github.com/foodcoops/foodsoft/blob/master/app/controllers/suppliers_controller.rb#L30 only allowing creation if at least one supplier_category exist. 
As there is no way to interact with supplier_category s as far as i can see i'd suggest setting this to be NULL in if there was now category selected.
Any thoughts on that?

PS: As a workaround i just unserted a default supplier category into the db.

**Comment by JuliusR** [comment]
Duplicate of #774 (fixed by #775, but not yet released).

As a workaround, you can open the `rails console` and type the following command (UNTESTED):
```
SupplierCategory.create!(name: "Other", financial_transaction_class: FinancialTransactionClass.first!)
```

**Comment by vali-um** [comment]
Sorry, i didn't see the closed issue and merged pull request... was only looking for open duplicates.
Apparently i mistakenly created my development database from the 4.7 release tag.
Thanks
